services/autoscaling/docker/entrypoint.sh

```diff
@@ -70,11 +70,6 @@ if [ "${SC_BUILD_TARGET}" = "development" ]; then
   fi
 fi
 
-if [ "${SC_BOOT_MODE}" = "debug" ]; then
-  # NOTE: production does NOT pre-installs debugpy
-  pip install --no-cache-dir debugpy
-fi
```

Comment:
Come to think more about it I think I added it here because I just updated to debugpy from a previous debugger which we used and that one was installed here... 🤷🏻‍♂️